Dockerfile

```
@@ -1,14 +1,44 @@
-
-FROM python:3.13
-WORKDIR /tmp
-COPY --from=builder /src/dist/*.whl .
-RUN pip3 install *.whl && rm *.whl
+FROM python:3.13-bookworm AS builder
+
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+
+WORKDIR /build
+
+ADD . /build/
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)